build/common.sh

```diff
@@ -97,35 +97,27 @@ kube::build::get_docker_wrapped_binaries() {
           kube-controller-manager,busybox
           kube-scheduler,busybox
           kube-proxy,gcr.io/google_containers/debian-iptables-amd64:v3
-          federation-apiserver,busybox
-          federation-controller-manager,busybox
```

Comment:
To confirm, the dev workflow of make a code change, run build/push-federation-images.sh and then run federation-up.sh again will still work right?